docs/examples_sequence.md

```diff
@@ -0,0 +1,53 @@
+# Toy Sequence Examples with Code
+
+## Preamble
+```python
+import numpy as np
+from pysr import *
+```
+
+Note that most of the functionality
+of PySRSequenceRegressor is inherited
+from PySRRegressor.
```

Comment:
Should link the docs pages using the mkdocs functionality to build references

Reply:
which page should it be linked to?

Reply:
ok for now i linked it to options.md, wdyt?